lib/stripe/card.ex

```diff
@@ -41,11 +43,43 @@ defmodule Stripe.Card do
     :tokenization_method
   ]
 
+  @response_mapping %{
+    id: :string,
+    address_city: :string,
+    address_country: :string,
+    address_line1: :string,
+    address_line1_check: :string,
+    address_line2: :string,
+    address_state: :string,
+    address_zip: :string,
+    address_zip_check: :string,
+    brand: :string,
+    country: :string,
+    customer: :string,
+    cvc_check: :string,
+    dynamic_last4: :string,
+    exp_month: :integer,
+    exp_year: :integer,
+    fingerprint: :string,
+    funding: :string,
+    last4: :string,
+    metadata: :metadata,
```

Comment:
The reason I chose `metadata` instead of map was to be explicit that it was just structureless data (from Stripe's perspective), and because `convert_value` has a `when is_map(format)` clause, which might make that confusing.